### PR TITLE
Feature: handle cairo1 `send_message_to_l1` syscall

### PR DESCRIPTION
**Pull request by maciejka**
Issue Number: N/A

## Type

- [x] feature
- [ ] bugfix
- [ ] dev (no functional changes, no API changes)
- [ ] fmt (formatting, renaming)
- [ ] build
- [ ] docs
- [ ] testing

## Breaking changes?

- [ ] yes
- [ ] no

### CODE REVIEW

File: tests/os.rs [hunk 1]
```diff
@@ -124,7 +124,21 @@ fn syscalls_cairo1(block_context: BlockContext, initial_state: InitialState, max
...
+
+    let test_send_message_to_l1_tx = test_utils::account_invoke_tx(invoke_tx_args! {
+        max_fee,
+        sender_address: sender_address,
+        calldata: create_calldata(contract_address, "test_send_message_to_l1", entrypoint_args),
+        version: tx_version,
+        nonce: nonce_manager.next(sender_address),
+    });
+
+    let txs = vec![test_emit_event_tx, test_storage_read_write_tx, test_get_block_hash_tx, test_send_message_to_l1_tx];
```
Comment (odesenfans) [inline tests/os.rs:141]: Unsure of how these tests work, but shouldn't there be at least an assertion somewhere to test that the code works?

**Review by odesenfans — APPROVED** [review]
Just one minor question, can be answered async.

File: tests/os.rs (hunk 1 above)
Comment (maciejka) [reply tests/os.rs:141]: You mean that each individual transaction succeeded?

Comment (odesenfans) [reply tests/os.rs:141]: Yes, and this does not not make sense in an integration test, thanks for the clarification.

**Pull request merged**